web/shared/components/page-header.tsx

```diff
@@ -46,6 +80,7 @@ export function PageHeader() {
                     <Dropdown.Menu className="bg-base-300 mt-4 z-10">
                         <Dropdown.Item onClick={handleOpenDebuggerPage}>Debugger</Dropdown.Item>
                         <Dropdown.Item onClick={handleOpenPlayerPage}>Player</Dropdown.Item>
+                        <Dropdown.Item onClick={handleOpenDashPage}>DASH Player</Dropdown.Item>
```

Comment:
There use in Live777 (liveion) and liveman both use this, But liveion don't offer this page

:) It's 404